.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,52 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: 'The name of the tarantool build artifact'
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: 'Clone the expirationd module'
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/expirationd
+
+      - name: 'Download the tarantool build artifact'
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+      # TODO(ylobankov): Just in case remove tarantool that could be previously
+      # installed on the runner. Relevant to self-hosted runners. Remove this
+      # step when only GitHub-hosted runners are used for integration testing.
+      - run: sudo apt-get -y purge 'tarantool*'
+      - name: 'Install tarantool'
+        # TODO(ylobankov): Install package dependencies. Now we're lucky: all
+        # dependencies are already there.
+        run: sudo dpkg -i tarantool*.deb
+
+      - name: 'Install expirationd'
```

Comment:
Could you specify in name that we install 'current' version of expirationd? `Install expirationd -current`

Reply:
Not relevant now.